### PR TITLE
[#136435555] Notifying tenants

### PR DESCRIPTION
**Pull request by saliceti**
# What

Story: [Improve incident comms](https://www.pivotaltracker.com/story/show/136435555)

Detailed instructions for notifying tenants, especially in case of an incident.

# How to review
Does it make sense?

# Who can review
Not me

### CODE REVIEW

**Comment by mtekel** [comment]
LG. Should someone else still review the email templates?

File: docs/team/notifying_tenants.md [hunk 1]
```diff
@@ -1,46 +1,112 @@
...
 
 * security problems and fixes
 * upgrades of CF, stemcells, buildpacks
 * incident reports
+* ongoing incidents
 
 ## Email draft
 
-Write the email you want to send.
+Write the email you want to send using a [template](#templates) as required and store it in [Google Docs](https://drive.google.com/drive/folders/0Bw4pWpR0IbJfWGFEMVBBZlFsSDQ).
```
Comment (urmishah) [inline docs/team/notifying_tenants.md]: Bit confusing. Make it clear you're sharing the draft in googledocs.

File: docs/team/notifying_tenants.md [hunk 2]
```diff
@@ -1,46 +1,112 @@
...
-for org in `cf orgs | tail -n+4`; do
-  cf target -o $org >/dev/null;
-  for space in `cf spaces 2>&1 | tail -n+4`; do
-    cf space-users $org $space;
-  done;
-done | grep '@' | grep -v 'Getting users in org' | sort | uniq | sed 's/^  //'
+Hello,
+
+We are aware of and investigating a problem with GOV.UK PaaS.
+[Summarise problem and scale]
```
Comment (urmishah) [inline docs/team/notifying_tenants.md]: There might be to much detail here which would delay sending. Its important to get something out fast here. Maybe make some of these optional or remove them from this template - they're in the update template.

File: docs/team/notifying_tenants.md [hunk 3]
```diff
@@ -1,46 +1,112 @@
...
+[Detail consequences for tenants AND end users]
+
+[Mention any action the user should take]
+[Mention things we know they should not do]
+[Explain workarounds for broken things]
+
+We will update you as soon as we know more information. In the meantime,
```
Comment (urmishah) [inline docs/team/notifying_tenants.md]: 'We will update you within the next 60 minutes'

As discussed in kickoff?

**Comment by saliceti** [comment]
Thanks, I've edited the doc

File: docs/team/notifying_tenants.md (hunk 3 above)
Comment (saliceti) [inline docs/team/notifying_tenants.md]: As discussed in real life, we want to keep updating the tenants hourly but it's better not to set the expectation too high. We've added instructions in the support manual instead.

**Pull request merged**